pkgs/test/test/runner/prefix_test.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+import 'package:test_core/src/util/prefix.dart';
+
+void main() {
+  test('testLibraryImportPrefix value is "test"', () {
+    expect(
+      testLibraryImportPrefix,
+      'test',
+      reason:
+          'testLibraryImportPrefix must be equal to the String "test". Some '
```

Comment:
We should call out exactly which tools, so that if we need to change it we know what to fix.